queries/fragments/rubickNft.graphql

```diff
@@ -0,0 +1,11 @@
+fragment rubickNft on NFTEntity {
```

Comment:
nah this deserves better name. (but it's not the focus of this PR)

Reply:
subsquidNft? ...

Reply:
Let's go with `subsquidNft` for now